modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/metadata/RelMetadataQueryEx.java

```diff
@@ -17,37 +17,14 @@
 
 package org.apache.ignite.internal.sql.engine.metadata;
 
-import static java.util.stream.Collectors.toList;
-import static org.apache.ignite.internal.util.IgniteUtils.igniteClassLoader;
-
-import io.github.classgraph.ClassGraph;
```

Comment:
can we remove the dependency at all?

Reply:
Thank you. It seems so. The dependency has been removed.